TACC/Core-CMS-Custom
# task/wp-346-submitter-reg-listing-permissions

## Overview

Allow users with APCD_ADMIN permissions to view registrations listing for submitter_admins without explicitly having a submitter_admin role.

## Related

- [WP-346](https://jira.tacc.utexas.edu/browse/WP-346)

## Changes

Changed dispatch of submitter_renewal_listings to check if user is a submitter_admin OR apcd_admin

## Testing

1. Go to [http://localhost:8000/admin](http://localhost:8000/admin)
2. Go to click users and your username
3. Remove submitter_admin from your groups and leave only APCD_ADMIN
4. Click save
5. Bring down your containers and bring them back up
6. Go to [http://localhost:8000/register/list-registration-requests/](http://localhost:8000/register/list-registration-requests/) and make sure you can still see your listings.
